spec/how_is_spec.rb

```diff
@@ -109,14 +109,14 @@
   end
 
   # Disable 'cop' that is violated by every .generate_frontmatter() calls.
-  # rubocop:disable Style/BracesAroundHashParameters
+  # rubocop:disable Layout/BracesAroundHashParameters
   context '#generate_frontmatter' do
     it 'works with frontmatter parameter using String keys, report_data using String keys' do
       actual = nil
       expected = nil
 
       VCR.use_cassette("how-is-example-repository") do
-        actual = HowIs.generate_frontmatter({'foo' => "bar %{baz}"}, {'baz' => "asdf"})
+        actual = HowIs.generate_frontmatter({'foo' => "bar %<baz>s"}, {'baz' => "asdf"})
```

Comment:
The published style guide has nothing on this in [its percent literals section](https://github.com/bbatsov/ruby-style-guide#percent-literals).

But the "cops" are a different set of things, and they have their own docs. 

I took the long way around, reading cops, and then I reset the repo, re-ran with the settings I noted in https://github.com/how-is/how_is/pull/175 

```
spec/how_is_spec.rb:131:59: C: Style/FormatStringToken: Prefer annotated tokens (like %<foo>s) over template tokens (like %{foo}).
        actual = HowIs.generate_frontmatter({:foo => "bar %{baz}"}, {:baz => "asdf"})
                                                          ^^^^^^
```